blueprints/ember-frost-css-core/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  description: '',
+  normalizeEntityName: function () {},
+
+  /**
+    Installs specified packages at the root level of the application.
+    Triggered by 'ember install <addon name>'.
+    @returns {Promise} package names and versions
+  */
+  afterInstall: function () {
+    return this.addAddonsToProject({
+      packages: [
+        {name: 'ember-frost-theme', target: '^1.0.2'}
```

Comment:
So, is this *instead of* or *in addition to* having the `dependency` in `package.json`?